composer.json

```diff
@@ -19,7 +19,7 @@
     ],
     "require": {
         "php": "^5.4 || ^7 || ^8",
-        "doctrine/dbal": "^2.5",
+        "doctrine/dbal": "^2.5 || ^3.1",
```

Comment:
Why not allow 3.0?
```suggestion
        "doctrine/dbal": "^2.5 || ^3.0",
```

Reply:
When adding support, I usually add the latest minor version, that's the only reason. I can allow `3.0`.

Reply:
IMO there's no reason to limit consumers of the lib not to use 3.0